sdk/src/wallet/operations/transaction/account.rs

```diff
@@ -27,51 +24,22 @@ where
     crate::client::Error: From<S::Error>,
 {
     /// Transitions an implicit account to an account.
-    pub async fn implicit_account_transition(
-        &self,
-        output_id: &OutputId,
-        key_source: Option<impl Into<BlockIssuerKeySource> + Send>,
-    ) -> Result<TransactionWithMetadata> {
+    pub async fn implicit_account_transition(&self, output_id: &OutputId) -> Result<TransactionWithMetadata> {
         let issuer_id = AccountId::from(output_id);
 
         self.sign_and_submit_transaction(
-            self.prepare_implicit_account_transition(output_id, key_source).await?,
+            self.prepare_implicit_account_transition(output_id).await?,
             issuer_id,
             None,
         )
         .await
     }
 
     /// Prepares to transition an implicit account to an account.
-    pub async fn prepare_implicit_account_transition(
-        &self,
-        output_id: &OutputId,
-        key_source: Option<impl Into<BlockIssuerKeySource> + Send>,
```

Comment:
Don't you think it would still be useful if users can set their own pubKeyHash?

Reply:
I reverted and changed a bit, not compiling yet but I had to go

Reply:
Fixed